Alencuben, Jack of All Trades Shuttle

Greenwall-Exe opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Added the Alencuben, a cubical slow moving shuttle with the ability to fill just about whatever role someone would want to fill throughout the shift. meant for at least 3 crewmates and is armed with a trio of rubicon emps and a pair of 20mm and 90mm cannons on the corners and their opposites, encouraging a spinning tactic.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
I came up with the idea based off borg cubes, but i wanted to see what a jack of all trades shuttle could be like, plus i wanted to see a shuttle with emp capabilities cause i've yet to actually see that in action so far. (think i might go make a fighter designed with rubicons after this)

## How to test
fly it about.

## Media
![image_2025-06-22_124934999](https://github.com/user-attachments/assets/4330704c-cdc7-48f2-a333-92ff103ae7a7)
## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ x] I have added media to this PR or it does not require an ingame showcase.
- [x ] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Alencuben
-->

Greenwall-Exe committed:
attempt#2

OneZerooo0 commented:
oh god pretty please make the tiling better, for example chemistry section is just orange… add more variance por favor

OneZerooo0 commented:
Also, starting blue space beaker, merc fab and sheer amount of gunnery… on a civilian craft is pretty dubious imo

Redrover1760 commented:
yaml unrelated, update your branch to get rid of it

Redrover1760 reviewed (changes requested):
Rubicons aren't allowed on non-faction ships according to current mapping guidelines. This also isn't a mil ship. I'd personally think a dedicated rubicon mil EMP ship considering the current meta might be fine but this isn't a mil ship or dedicated to emps.

your guns have basically no firing lines, 20mms blocking the way of 90mms. I'd personally recommend just only having 20mms 

Ship has an excessive amount of empty space in rooms that aren't meant to be empty, entire rooms with no touch of decals or flooring to accentuate the rooms I.e. med/sci/chem, empty tables, etc

lacks at least a basic kitchen type setup, a small wall-mount freezer at the least is practically required for a ship this size

150k is too low for a true jack of all trades ship but we can worry about that later.

Should be Catagory: Large instead of Medium in Resources/Prototypes/_Mono/Shipyard/Alencuben.yml

We don't really have a catagory of ship that would fit for a true jack of all trades, you're either exped, civilian, or military.

overall, cool concept, could use work. See me after class.

also this is the sorta ship that'd need like 5 people minimum to use properly.

Doesn't really have a focus, I know that's sorta the point but what's the usecase of this ship or its purpose?

Greenwall-Exe commented:
> oh god pretty please make the tiling better, for example chemistry section is just orange… add more variance por favor
> Also, starting blue space beaker, merc fab and sheer amount of gunnery… on a civilian craft is pretty dubious imo

fair enough, an interior decorator i am not, but will see what i can do. thanks for the feedback.

EctoplasmIsGood commented:
> > oh god pretty please make the tiling better, for example chemistry section is just orange… add more variance por favor
> > Also, starting blue space beaker, merc fab and sheer amount of gunnery… on a civilian craft is pretty dubious imo
> 
> fair enough, an interior decorator i am not, but will see what i can do. thanks for the feedback.

Remember to post updated pictures when you change it

Greenwall-Exe commented:
> Rubicons aren't allowed on non-faction ships according to current mapping guidelines. This also isn't a mil ship. I'd personally think a dedicated rubicon mil EMP ship considering the current meta might be fine but this isn't a mil ship or dedicated to emps.
> 
> your guns have basically no firing lines, 20mms blocking the way of 90mms. I'd personally recommend just only having 20mms
> 
> Ship has an excessive amount of empty space in rooms that aren't meant to be empty, entire rooms with no touch of decals or flooring to accentuate the rooms I.e. med/sci/chem, empty tables, etc
> 
> lacks at least a basic kitchen type setup, a small wall-mount freezer at the least is practically required for a ship this size
> 
> 150k is too low for a true jack of all trades ship but we can worry about that later.
> 
> Should be Catagory: Large instead of Medium in Resources/Prototypes/_Mono/Shipyard/Alencuben.yml
> 
> We don't really have a catagory of ship that would fit for a true jack of all trades, you're either exped, civilian, or military.
> 
> overall, cool concept, could use work. See me after class.
> 
> also this is the sorta ship that'd need like 5 people minimum to use properly.
> 
> Doesn't really have a focus, I know that's sorta the point but what's the usecase of this ship or its purpose?

1. ah dang, the origin of the idea was the borg cube and i havent seen any form of EMP on any other shuttle i've come across. you'd have emps to disable shuttles since its slow as hell and cannons to turn to to do actual damage. 
2. The firing lines are supposed to be kinda shit by design so that you can only have at most 2 corners firing on a target, one of which being emps. wanted to encourage a spinning attack, but if the rubicons gotta get nixed then different arrangement needs to be considered.
3. Wasn't quite certain how much things just lying about were cosher, or what exactly to put in there but i'll see what i can do to fill the space, i have a few ideas now that im awake again and thinkin on it
4. I think i can fit something in the bar, maybe a fridge and a stove if i drop one of the chairs and add a grill
5. the appraisal price was like 110000, so i took it up to 150000 as a filler, so fair enoug there.
6. again fair enough, wasnt sure if it qualified cause ive seen much bigger shuttles like the flashpoint
7. i thought of making it exped capable but then realized i didnt know how to do that and i had like...20 minutes left in my day and my brain was fried. 
8. Yes teach.
9. definitely intended to be manned by a crew rather than a single person
10. primarily i saw it as a shuttle capable of using pretty much every bit of wreckage chunks, artifacts, chem barrels, materials, animal bits etc. in hind sight i also realized there's no ore refiners to make use of mining asteroids. we did have that grand honker shuttle that had a full sized kitchen, hydroponics, robotics, manufacturing, chemlab, armory, a temple, secret room of death that had clown spiders in it and an artillery piece specifically to flashbang an audience at a grand stage.

mikusssssss commented:
bro what in the living fuck is this ame holy shit

sleepyyapril commented:
please don't keep the ame like that, you can fit three cores NORMALLY without resorting to a war crime

also update branch to fix test fail, it's not your fault

ScyronX commented:
This is more in the category of a mini station rather then a shuttle

gameandglitch33 commented:
>> we did have that grand honker shuttle that had a full sized kitchen, hydroponics, robotics, manufacturing, chemlab, armory, a temple, secret room of death that had clown spiders in it and an artillery piece specifically to flashbang an audience at a grand stage.

*Distant sobbing*

Redrover1760 commented:
for some of that empty space you may wanna add posters and stuff, other methods of covering deadzone via wallmount stuff, a fire axe, station stuff basically

Greenwall-Exe committed:
updated it with aforementioned notes in mind, sans the ame, i like how it fits.

Redrover1760 reviewed (changes requested):
you need to fix your changelog and update the image of your ship on the pr body btw. Sorry its been a bit, was busy

Merging main into your branch will fix the reuse failing check

mikusssssss commented:
hey buddy… man who fixes the changelog here… i’m here to inform you to… fix your changelog…

Greenwall-Exe commented:
oop my bad meant to work on this over the weekend but got completely side tracked. Last thing i did was update the branch...weird. Ill get it fixed up when i get home in the morning

Greenwall-Exe committed:
updated, filled empty space changed armaments etc.

Greenwall-Exe commented:
![Screenshot 2025-06-30 070840](https://github.com/user-attachments/assets/919b33e1-d9f2-446e-8f1d-95d8559ae5ae)
![Screenshot 2025-06-30 070954](https://github.com/user-attachments/assets/10fc4602-75ed-4290-8eea-32dea0af7e24)
![Screenshot 2025-06-30 071045](https://github.com/user-attachments/assets/06cabab1-483d-4f65-a964-9ffb9c6d2057)
![Screenshot 2025-06-30 071154](https://github.com/user-attachments/assets/2e6a5362-ba9e-45f9-a180-0fbf3dffab6b)
![Screenshot 2025-06-30 071230](https://github.com/user-attachments/assets/8c1239af-6219-4389-b223-3d5e72704e4c)

Greenwall-Exe commented:
i just noticed the random steak in the corner...where the shit did that come from

Greenwall-Exe committed:
removed wild fax machine steak

Greenwall-Exe commented:
:cl:
- added Lots of stuff to fill space
- added atmos mining equipment
- added the beginning of a clarke
- added stocked bar and freezer and added a cooking range 
- removed the wild fax machine steak
-->

DingleFlop commented:
I ALSO have many concerns about the balance of this vessel.
a 25s jump drive is 75k ON ITS OWN, you quoted this ship at 150k but as it stands I don't think 250k is even a fair price.

Rule of thumb: If it's usually something you need to go to TO or CO to pick up, you probably shouldn't include it on your ship.

It's okay to have interesting things on your ship (Like that AME, I actually think is really cool despite the dissenters!) but you can't just give yourself OP shit 4no'raisins.

crazy1112345 commented:
> :cl:
> - added Lots of stuff to fill space
> - added atmos mining equipment
> - added the beginning of a clarke
> - added stocked bar and freezer and added a cooking range 
> - removed the wild fax machine steak
> -->
> 
Atmos mining stuff is good, but it needs an external gaslock on the shuttle somewhere that can actually reach the portable one. This is usually done with a probe (see akasano or the arribane i PR'd), if its flush with the long flat wall it can be essentially impossible to actually dock with the mining setup, so you may have to break the cube a little or figure something else out.

BlueHNT commented:
I don't like it.

Jack of all trades ships are horrendous. Not to mention it being a literal cube... 

A trading vessel being cube like is fine... But a literal cube trying to do everything is horrible.

I urge @starch70 to make some comments on this before merging it.

Redrover1760 commented:
> 🆑
> 
>     * added Lots of stuff to fill space
> 
>     * added atmos mining equipment
> 
>     * added the beginning of a clarke
> 
>     * added stocked bar and freezer and added a cooking range
> 
>     * removed the wild fax machine steak
>       -->

Please update your screenshots on the main PR body
